### ISSUE TITLE
typo in how to build non master branches

### ISSUE THREAD

**Issue by cartazio** [ghcjs/ghcjs#582]
it seems to say

```
$ ghcjs-boot --dev --ghcjs-boot-dev-branch somebranch --shims-dev-branch
```

but I think it should say 

```
$ ghcjs-boot --dev --ghcjs-boot-dev-branch somebranch --shims-dev-branch somebranch
```
?

**Comment by philderbeast** [comment]
I noticed that too. On #595 I add the `shims` branch name with a concrete example of `ghc-8.0`.

**Comment by hamishmack** [comment]
Fixed by #595
